### PR TITLE
edit README.md

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**

### CODE REVIEW

**Review by caleb-at-pieces — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,32 +9,53 @@
 Please make sure you have [**Pieces OS**](https://docs.pieces.app/installation-getting-started/what-am-i-installing) installed to run the Package.
 
 
-## Getting Started
```
Comment (shivay-at-pieces) [inline README.md:12]: The Pieces Sublime Plugin logo should have the image similar to other plugins: https://github.com/pieces-app/obsidian-pieces/raw/main/assets/readme/pfd-obisidan-plugin-hero.png

**Review by shivay-at-pieces — CHANGES_REQUESTED** [review]
The starting

File: README.md (hunk 1 above)
Comment (shivay-at-pieces) [reply README.md:12]: @laurin-at-pieces can provide the image for Sublime plugin

File: README.md [hunk 2]
```diff
@@ -9,32 +9,53 @@
...
-If you're viewing a saved material, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
+Want to make changes to your saved materials? Switch to editor mode by clicking the **Edit** button on your saved snippet. Modify the content as needed and save your changes with a simple `command/ctrl + s`. It's that easy!
 
-### Right click to add an asset
+### Shareable Link
 
-To save a new snippet in Pieces, select the code you wish to save, right-click to open the context menu, and select `Pieces > Save to Pieces`.
+Ever wanted to share a code snippet effortlessly? Select the code, right-click to open the context menu, and choose `Pieces > Generate shareable link`. You can also share a saved material by opening it and clicking the **Share** button. Sharing has never been simpler!
+
+### Right Click to Add an Asset
```
Comment (mack-at-pieces) [inline README.md:43]: "Right Click to Save to Pieces"

Comment (mack-at-pieces) [reply README.md:43]: Can we make this one first in the order AFTER "Configuration"

File: README.md [hunk 3]
```diff
@@ -9,32 +9,53 @@
...
+
+Saving a new snippet in Pieces is a breeze! Just select the code you want to save, right-click to open the context menu, and choose `Pieces > Save to Pieces`. Your code is now safely stored!
 
 ### Copilot Ask
```
Comment (mack-at-pieces) [inline README.md:47]: Can we make this one second in the order after "Save to Pieces"

File: README.md [hunk 4]
```diff
@@ -9,32 +9,53 @@
...
 
-Need the Copilot to comment, fix a bug, or add a doc string? Select some code, right-click to open the context menu, and select Pieces > Ask Copilot.
+Need some help from Copilot? Whether it's commenting, fixing a bug, or adding a docstring, just select the code, right-click to open the context menu, and choose `Pieces > Ask Copilot`. Let Copilot do the heavy lifting for you!
+
+
+### Open a Copilot Conversation
```
Comment (mack-at-pieces) [inline README.md:52]: This one can be third

File: README.md [hunk 5]
```diff
@@ -9,32 +9,53 @@
...
+
+Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Simply open the command palette and select `Pieces: Copilot`. From there, you can ask anything whether it's for coding help, debugging, or even generating snippets to save to Pieces with some clicks.
```
Comment (mack-at-pieces) [inline README.md:54]: I think we should move this copy, "Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips." into the "Copilot Ask" section.

"Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Whether it's commenting, fixing a bug, or adding a docstring, just select the code, right-click to open the context menu, and choose `Pieces > Ask Copilot`. Let Copilot do the heavy lifting for you!"

Comment (mack-at-pieces) [reply README.md:54]: Then this one can be more oriented around viewing their existing conversations.. 

"Need to open an existing Copilot Chat, or start a new one? Simply open the command palette and select `Pieces: Copilot`. ..."

File: README.md (hunk 3 above)
Comment (mack-at-pieces) [inline README.md:47]: "Engage with the Pieces Copilot"

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]

**Comment by Bishoy-at-pieces** [comment]
@mack-at-pieces seems I did not explain well the difference between the "Ask" and the "Copilot Ask" 
the ask from the context menu I think I need better description for the "Copilot Ask"
image

and can you suggest me better name?

File: README.md [hunk 6]
```diff
@@ -9,32 +9,61 @@
 Please make sure you have [**Pieces OS**](https://docs.pieces.app/installation-getting-started/what-am-i-installing) installed to run the Package.
 
 
-## Getting Started
+## Getting Started with Pieces Sublime Plugin
```
Comment (mack-at-pieces) [inline README.md:12]: "Getting Started with the Pieces Sublime Plugin"

File: README.md [hunk 7]
```diff
@@ -9,32 +9,61 @@
...
+![](https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/ask%20stream.gif)
 
 ### Copilot Ask
 
-Need the Copilot to comment, fix a bug, or add a doc string? Select some code, right-click to open the context menu, and select Pieces > Ask Copilot.
+Need some help from Copilot? Whether it's commenting, fixing a bug, or adding a docstring, just select the code, right-click to open the context menu, and choose `Pieces > Ask Copilot`. Let Copilot do the heavy lifting for you!
+
+![](https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/ask.gif)
+
+### Open Saved Material
```
Comment (mack-at-pieces) [inline README.md:42]: "Open Saved Materials"

File: README.md [hunk 8]
```diff
@@ -9,32 +9,61 @@
...
-### Right click to add an asset
+Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Simply open the command palette and select `Pieces: Copilot`. From there, you can ask anything whether it's for coding help, debugging, or even generating snippets to save to Pieces with some clicks.
 
-To save a new snippet in Pieces, select the code you wish to save, right-click to open the context menu, and select `Pieces > Save to Pieces`.
+![](https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/ask%20stream.gif)
 
 ### Copilot Ask
 
-Need the Copilot to comment, fix a bug, or add a doc string? Select some code, right-click to open the context menu, and select Pieces > Ask Copilot.
+Need some help from Copilot? Whether it's commenting, fixing a bug, or adding a docstring, just select the code, right-click to open the context menu, and choose `Pieces > Ask Copilot`. Let Copilot do the heavy lifting for you!
```
Comment (mack-at-pieces) [inline README.md:38]: "Need some help from the Pieces Copilot? ..."

File: README.md [hunk 9]
```diff
@@ -9,32 +9,61 @@
...
-If you're viewing a saved material, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
+### Engage with the Pieces Copilot
 
-### Right click to add an asset
+Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Simply open the command palette and select `Pieces: Copilot`. From there, you can ask anything whether it's for coding help, debugging, or even generating snippets to save to Pieces with some clicks.
 
-To save a new snippet in Pieces, select the code you wish to save, right-click to open the context menu, and select `Pieces > Save to Pieces`.
+![](https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/ask%20stream.gif)
 
 ### Copilot Ask
```
Comment (mack-at-pieces) [inline README.md:36]: "Use the Pieces Copilot to Modify Code or Solve Problems"

File: README.md [hunk 10]
```diff
@@ -9,32 +9,61 @@
 Please make sure you have [**Pieces OS**](https://docs.pieces.app/installation-getting-started/what-am-i-installing) installed to run the Package.
 
 
-## Getting Started
+## Getting Started with the Pieces Sublime Plugin
```
Comment (shivay-at-pieces) [inline README.md:12]: Hey @Bishoy-at-pieces so keeping the readme streamlined like the other readmes for our plugins, 

You can start the readme this way: 
```
# [Pieces for Developers](https://pieces.app) | Sublime Plugin

Then the hero image 

## Overview 
What is the Sublime plugin

## Features

<!--   Add all of the features here --> 
```

**Review by shivay-at-pieces — CHANGES_REQUESTED** [review]

**Review by shivay-at-pieces — APPROVED** [review]

**Review by mack-at-pieces — APPROVED** [review]

**Pull request merged**